student.rb

```diff
@@ -1,9 +1,11 @@
 require './person'
 
 class Student < Person
-  def initialize(id, age: 0, parent_permission: true, classroom: 'Unknown', name: 'Unknown')
+  attr_accessor :classroom, :name
+
+  def initialize(id, age: 0, parent_permission: true, name: 'Unknown')
     super(id, name, age, parent_permission)
-    @classroom = classroom
+    @classroom = nil
   end
```

Comment:
- Great JOB. In reading the requirements, it says: "Make sure that when setting the classroom for a student it also adds it to the classrooms' students". This means that you need to add a [method that serves as the `belongs-to`](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)  side (For this case `a student belongs to a classroom`)
- Please click on the blue link above to understand well
```ruby
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```